echo-core/src/main/java/com/netflix/spinnaker/echo/artifacts/MessageArtifactTranslator.java

```diff
@@ -56,7 +58,7 @@ public MessageArtifactTranslator(InputStream templateStream, JinjavaFactory jinj
       this.jinjaTemplate = "";
     } else {
       try {
-        this.jinjaTemplate = IOUtils.toString(templateStream);
+        this.jinjaTemplate = IOUtils.toString(templateStream, Charset.defaultCharset());
```

Comment:
I don't feel super comfortable with `defaultCharset()`, can we explicitly declare `Charsets.UTF_8` instead?

Reply:
I actually wanted to make it `Charsets.UTF_8` but the deprecated one-argument method itself calls `Charset.default()` for the missing argument so this seemed more backwards compatible.

That being said, it's probably safer and better to just enforce UTF-8 here so I can make the change.